client/policy_validator.go

```diff
@@ -300,6 +333,19 @@ type PolicyAPIResult struct {
 	Error     string   `json:"error"`
 }
 
+type allowedBy map[string][]string
```

Comment:
Seems odd to me to have an unexported type for exported fields on an exported struct. Is that deliberate?

Reply:
Intention was just to prevent it to be used externally, but I agree it feels odd.
I will make it an exported type.